### PR TITLE
Combination of Quick-upgrade + Upgrade able view

### PR DESCRIPTION
**Pull request by Stx69**
Dear DF community, 
glad to try what I imagine. Long time was not active here and I don't know if I will managed all right for smootly release. Let's try...

Any feedback welcome..

### CODE REVIEW

**Comment by jacobrosenthal** [comment]
I think were having trouble communicating and this is kind of a mess now :( . The point of making a new plugin would be not to touch the old plugin in case someone likes it. You would want to do one or the other.

File: content/productivity/quick-view-upgradeable/plugin.js [hunk 1]
```diff
@@ -0,0 +1,364 @@
+// Quick View Upgradeable
+//
+// Combination of Quick-upgrade + Upgrade able view plugins
```
Comment (Stx69) [inline content/productivity/quick-view-upgradeable/plugin.js:3]: test suggestion :D

File: content/productivity/quick-view-upgradeable/plugin.js [hunk 2]
```diff
@@ -0,0 +1,364 @@
...
+
```
Comment (Stx69) [inline content/productivity/quick-view-upgradeable/plugin.js:4]: add 2

**Review by Stx69 — COMMENTED** [review]
suggest for description

**Comment by Stx69** [comment]
Closed. was first and bad.... next one will be better I hope.

**Pull request closed without merging**